gen/calc.py

```diff
@@ -919,6 +919,7 @@ def node_check_names(config):
         'mesos_log_retention_mb': '4000',
         'mesos_container_log_sink': 'logrotate',
         'mesos_max_completed_tasks_per_framework': '',
+        'mesos_recovery_timeout': '24hrs',
```

Comment:
Can we get a validate function on this input, so users don't accidentally try "24hours" or "24h" or "24" or "1day" or something else that won't be parsed properly by Mesos?

Reply:
@adam-mesos Sure, I just pushed a new commit that adds validation.